omnigibson/robots/behavior_robot.py

```diff
@@ -5,6 +5,7 @@
 import os
 from scipy.spatial.transform import Rotation as R
 from typing import List, Tuple, Iterable
+from telemoma.human_interface.teleop_core import TeleopAction
```

Comment:
Don't import this here. Creates an unnecessary dependency. Remove the type hint it's used for.